pom.xml

```diff
@@ -4,13 +4,18 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>net.sf</groupId>
+    <groupId>livongo</groupId>
     <artifactId>log4jdbc4</artifactId>
-    <version>1.3</version>
+    <version>1.3.1</version>
```

Comment:
I don't know if we should be changing the organization (although I am not opposed) since we have forked. Since we are changing the org, should we reset the version?

**`[food-for-thought]`**

Reply:
I don't know either. The change was made because we use `log4jdbc4` [published with `livongo` as organisation](https://artifactory.internal.livongo.com/ui/native/ext-release-local/livongo/log4jdbc4). But I agree we should respect the original author, so I'll revert back to the original org and leave the version `1.3.1`.

Reply:
No preference